lib/omniauth/strategies/openid_connect.rb

```diff
@@ -143,10 +158,17 @@ def issuer
       end
 
       def discover!
-        client_options.authorization_endpoint = config.authorization_endpoint
-        client_options.token_endpoint = config.token_endpoint
-        client_options.userinfo_endpoint = config.userinfo_endpoint
-        client_options.jwks_uri = config.jwks_uri
+        return unless options.discovery
+        options.issuer = issuer if options.issuer.blank?
```

Comment:
Am i right in assumption `issuer` won't be set if `discovery` disabled?

Reply:
Yes.
Discovery `issuer` will be only used only `discovery` is enabled and options `issuer` was `blank?`

Another point, I'm wondering if you prefer that I avoid using rails `blank?` method here.

Reply:
> Yes.
> Discovery issuer will be only used only discovery is enabled and options issuer was blank?

Is this intentional change? Old code has different behaviour, I'm not sure about this.

> Another point, I'm wondering if you prefer that I avoid using rails blank? method here.

Yeah, good point, it'd be great to avoid rails' methods.